### PR TITLE
Changed team members and Chairman images and information

### PR DESCRIPTION
**Pull request by Keshav7104**
###Changed team members and Chairman images and information
##
Screenshot 2023-12-21 065135
#also changed all of their social media handles
Screenshot 2023-12-21 065101

### CODE REVIEW

**Comment by tripathics** [comment]
@Keshav7104 can you add a small section for past editors? It should not use these card components

**Comment by tripathics** [comment]
Just checked the preview, Omsubhra pic is not visible (maybe fix the src). And Aryan's pic is not fit for that page.

File: app/(pages)/about/page.js [hunk 1]
```diff
@@ -57,51 +57,60 @@ const People = {
...
     {
-      name: 'Daknya Bam',
-      role: 'Front-end, UI-UX',
+      name: 'Omsubhra Singha',
+      role: 'Designer and Editor',
       designation: 'UG Student, CSE 3rd year',
-      description: 'A coder. singer,guitarist a full pack of combination of computer science undergratuate',
-      profile: 'https://avatars.githubusercontent.com/u/75485331',
+      description: 'None',
+      profile: 'https://scontent.fmaa8-1.fna.fbcdn.net/v/t39.30808-6/343521559_1654178948388917_6796477082981265128_n.jpg?stp=c0.62.526.526a_dst-jpg_p526x296&_nc_cat=108&ccb=1-7&_nc_sid=5f2048&_nc_ohc=hsGs_emVZ_gAX-ub6jU&_nc_ht=scontent.fmaa8-1.fna&oh=00_AfBIOikf-Nf3axcsDGElY1OlIMRlgejzCDxZJYl5b9oAuw&oe=65486239',
```
Comment (dev-singh-kanyal) [inline app/(pages)/about/page.js:88]: @Keshav7104 correct the profile link of Omsubra

Comment (dev-singh-kanyal) [reply app/(pages)/about/page.js:88]: or upload it directly in `/images/`

**Comment by Keshav7104** [comment]
> @Keshav7104 can you add a small section for past editors? It should not use these card components

Ok i will look for that part in a separate pull request

**Comment by tripathics** [comment]
> > @Keshav7104 can you add a small section for past editors? It should not use these card components
> 
> Ok i will look for that part in a separate pull request

Infact, create a new issue for this one so that others can also work on it.

File: next.config.js
Comment (dev-singh-kanyal) [inline next.config.js:1]: So, this change removes the redundant domains from which it can load images. @Keshav7104 👍🏻

File: app/(pages)/about/page.js
Comment (dev-singh-kanyal) [inline app/(pages)/about/page.js:1]: Profile picture link from an external URL to a local file path and shortened `Aaryan raj` to `Aaryan`. @Keshav7104 👍🏻

**Review by dev-singh-kanyal — APPROVED** [review]
Great work on updating the team members and chairman images and information. Everything looks good to me. I approve this pull request and will merge it now.

**Pull request merged**